src/__main__.py

```diff
@@ -41,6 +43,8 @@
     """
     Initializes and connects the client, then passes it to rat_command.
     """
+
+    config = setup(cli_manager.GET_ARGUMENTS().config_file)
```

Comment:
[question]  should i move `setup()` to something a tad more descriptive, like `reload_configuration` ?